README.md

```diff
@@ -1,2 +1,30 @@
-# cumulus-library-suicidality-icd10
-Validation of ICD-10 code usage of Suicidality Cases Compared to Patient Chart Review
+# How reliable are ICD-10 diagnosis codes to identify cases of pediatric suicidality?
+
+This study repository includes:
+
+* **Suicidality ICD10 case definition** (SQL on FHIR) 
+* [Propensity Score Matching](https://github.com/smart-on-fhir/cumulus-library/blob/main/tests/test_psm.py) (PSM) to select "case matched comparators"
+* [Chart Review](https://docs.smarthealthit.org/cumulus/chart-review/) software for reproducing results
```

Comment:
this is not actually in the repo - perhaps move this to a seperate list of related software? or link to the supplement doc

Reply:
This = Chart Review? Ok I'll make a related packages ....